src/ReactNativeDeviceActivityModule.ios.ts

```diff
@@ -1,13 +1,28 @@
 import { requireNativeModule } from "expo-modules-core";
 
 import { appGroupName } from "./AppGroup";
+import { ReactNativeDeviceActivityNativeModule } from "./ReactNativeDeviceActivity.types";
 
-// It loads the native module object from the JSI or falls back to
-// the bridge module (from NativeModulesProxy) if the remote debugger is on.
-const deviceActivityModule = requireNativeModule("ReactNativeDeviceActivity");
+const loadDeviceActivityModuleSafe = ():
+  | ReactNativeDeviceActivityNativeModule
+  | undefined => {
+  try {
+    // It loads the native module object from the JSI or falls back to
+    // the bridge module (from NativeModulesProxy) if the remote debugger is on.
```

Comment:
I'm not sure if this comment is still accurate. I think this refers to old remote debugging, so probably doesn't make sense with modern react-native versions

Reply:
Good one!